<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fieldNavigationClass</class>
 <widget class="QMainWindow" name="fieldNavigationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ProcessingTimeLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>279</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ProcessingTime </string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetTop" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(169, 201, 226);</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>2</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Field navigation at crop seedling stage</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetBottom" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>39</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 247, 250);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Resolution selection</string>
     </property>
    </widget>
    <widget class="QComboBox" name="resolutionComboBox">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>48</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(195, 195, 195);</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(195, 195, 195);</string>
     </property>
     <property name="text">
      <string>Processing</string>
     </property>
     <property name="icon">
      <iconset theme="document-open"/>
     </property>
    </widget>
   </widget>
   <zorder>widgetBottom</zorder>
   <zorder>widgetTop</zorder>
   <zorder>ProcessingTimeLabel</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fieldNavigationProject.qrc"/>
 </resources>
 <connections/>
</ui>
